src/Microsoft.SqlTools.ServiceLayer/Migration/MigrationService.cs

```diff
@@ -402,6 +405,50 @@ public void InitializeService(ServiceHost serviceHost)
             }
         }
 
+        internal async Task HandleSaveSkuRecommendationsRequest(
+            SaveSkuRecommendationsParams parameters,
+            RequestContext<SaveSkuRecommendationsResult> requestContext)
+        {
+            const string SkuRecommendationJsonFileExt = ".json";
+            const string SkuRecommendationHtmlFileExt = ".html";
+            string timeStamp = DateTime.UtcNow.ToString("yyyyMMddHHmmss");
```

Comment:
we can skip ss but mm we need in case customer wants to save multiple sku recommendation files while data collection is still going on. And customer can do this if they want to see how (or if) recommendation varies with time.